src/main/java/com/netflix/imflibrary/st0377/header/StructuralMetadata.java

```diff
@@ -705,16 +712,22 @@ else if (isStructuralMetadata(key) && (key[13] == 0x01))
                     return GroupOfSoundFieldGroupLabelSubDescriptor.GroupOfSoundFieldGroupLabelSubDescriptorBO.class;
                 case 0x5A :
                     return JPEG2000PictureSubDescriptor.JPEG2000PictureSubDescriptorBO.class;
+                case 0x7B :
+                    return IABEssenceDescriptor.IABEssenceDescriptorBO.class;
+                case 0x7C :
+                    return IABSoundfieldLabelSubDescriptor.IABSoundfieldLabelSubDescriptorBO.class;
                 case 0x79 :
                     return ACESPictureSubDescriptor.ACESPictureSubDescriptorBO.class;
                 case 0x7a :
                     return TargetFrameSubDescriptor.TargetFrameSubDescriptorBO.class;
                 default :
+                    System.out.printf("Unknown value for key[14]: "+key[14]+"\n");
```

Comment:
We should use logger instead.

Reply:
This is a left-over from some initial debug. I removed it.